.github/workflows/cicd.yml

```
@@ -1,15 +1,34 @@
+  pull_request:
+    branches:
+      - main
   push:
     branches:
       - "*"
 
 jobs:
+  trunk-check:
+    uses: NextGenContributions/cicd-pipeline/.github/workflows/trunk-check.yml@main
+    permissions:
+      checks: write # For trunk to post annotations
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)